### PR TITLE
Added DiscordJS Guide & added Font Awesome

### PR DESCRIPTION
**Pull request by OGMatrix**
**Font Awesome on Main:**
ss1

**Font Awesome on viewing a Icon:**
ss2

**DiscordJS Guide on Page:**
ss1
ss2

**If you need more Images just say it!**

### CODE REVIEW

**Comment by EncryptedDev** [comment]
The Disocrd.js one literally just takes a H1... low quality presence?

**Comment by OGMatrix** [comment]
> The Disocrd.js one literally just takes a H1... low quality presence?

But its a presence

**Comment by EncryptedDev** [comment]
> > The Disocrd.js one literally just takes a H1... low quality presence?
> 
> But its a presence

Maybe instead support Discord.js' full website instead of just a niche part of the website

**Comment by OGMatrix** [comment]
> It's all pretty low quality 🤷

It doesnt go on the Quality, it goes on the usages

File: websites/D/Discord BotList/dist/metadata.json [hunk 1]
```diff
@@ -24,5 +24,19 @@
...
+        "value": true
+      },
+      {
+        "id": "submit",
+        "title": "Show Submit",
+        "icon": "fas fa-plus-circle",
+        "value": true
+      }
+    ]
   }
```
Comment (Bas950) [inline websites/D/Discord BotList/dist/metadata.json:42]: bump version

File: websites/D/DiscordJS.guide/presence.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+pPresence.on("UpdateData", async() => {
+  const pData: PresenceData = {
+    largeImageKey: "icon2"
+  },
+    pPage = window.location.pathname;
+
+  pData.startTimestamp = pBrowsing;
+
+  if (pPage.includes("/")) {
```
Comment (Bas950) [inline websites/D/DiscordJS.guide/presence.ts:14]: always true

File: websites/D/DiscordJS.guide/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "DiscordJS Guide",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/lJOT2LO.png",
+    "thumbnail": "https://i.imgur.com/ojmsp7f.png",
```
Comment (Bas950) [inline websites/D/DiscordJS.guide/dist/metadata.json:10]: logo needs to be at least 512x512

File: websites/D/DiscordJS.guide/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
+
```
Comment (Bas950) [inline websites/D/DiscordJS.guide/tsconfig.json:7]: remove extra line

File: websites/F/Font Awesome/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "FontAwesome",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/Xh9923z.png",
+    "thumbnail": "https://i.imgur.com/GJFmAgg.png",
```
Comment (Bas950) [inline websites/F/Font Awesome/dist/metadata.json:10]: logo needs to be at least 512x512

File: websites/F/Font Awesome/presence.ts [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    pData.state = "Main";
+  } else if (pPage.includes("/icons")) {
+    const urlParams: URLSearchParams = new URLSearchParams(window.location.search),
+      myParam: string = urlParams.get('q');
+
+    if (myParam == null) {
+      try {
+        const icon: string = document.querySelector("#page-top > div.view.flex.flex-column.min-vh-100.db-pr > div.flex-grow-1.flex-shrink-0.flex-basis-auto > div > div.ph6-l > div > section > header > div.flex.flex-column.flex-row-xl.items-center-xl.justify-between-xl.mb2.mb4-l > h1 > span").textContent;
+        pData.details = "Watching Icon:";
+        pData.state = icon;
```
Comment (Bas950) [inline websites/F/Font Awesome/presence.ts:25]: Viewing icon not watching

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
First of all fix:
image

**Comment by OGMatrix** [comment]
> First of all fix:
> image

@Bas950 I dont get whats wrong

**Comment by Bas950** [comment]
> > First of all fix:
> > image
> 
> @Bas950 I dont get whats wrong

you probs just need to update your fork

**Comment by Bas950** [comment]
> > > > First of all fix:
> > > > image
> > > 
> > > 
> > > @Bas950 I dont get whats wrong
> > 
> > 
> > you probs just need to update your fork
> 
> Did it doesnt work

image
you sure about that?

File: websites/D/DiscordJS.guide/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "DiscordJS Guide",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/Zt4i2l3.png",
+    "thumbnail": "https://i.imgur.com/ojmsp7f.png",
+    "version": "1.0.1",
```
Comment (callumok2004) [inline websites/D/DiscordJS.guide/dist/metadata.json:11]: New presence, use version 1.0.0

File: websites/F/Font Awesome/dist/metadata.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "FontAwesome",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/SnQ9HYz.png",
+    "thumbnail": "https://i.imgur.com/GJFmAgg.png",
+    "version": "1.0.1",
```
Comment (callumok2004) [inline websites/F/Font Awesome/dist/metadata.json:11]: New presence, use version 1.0.0

**Comment by Bas950** [comment]
Fix merge conflicts

**Comment by OGMatrix** [comment]
> Fix merge conflicts

how

**Comment by OGMatrix** [comment]
new pr

**Pull request closed without merging**